_posts/2022-07-15-bottlerocket-k8s-fluent-bit.markdown

```diff
@@ -0,0 +1,401 @@
+---
+layout: post
+title:  "How to use Fluent Bit and OpenSearch with Bottlerocket and Kubelet logs"
+authors:
+  - kyledvs
+date:   2022-07-15
+categories:
+  - technical-post
+excerpt: "Today, I’m going to show you how to run OpenSearch with the OpenSearch Operator on Kubernetes using the Bottlerocket then add Fluent Bit to collect logs from the nodes of the same Kubernetes cluster."
+---
+
+This place seems familiar 👀
+
+Today, I’m going to show you how to run OpenSearch with the OpenSearch Operator on Kubernetes using the Bottlerocket then add Fluent Bit to collect logs from the nodes of the same Kubernetes cluster. Phew. If that seems like a lot, don’t worry - this post will take it bit-by-bit. I’ll assume you know what OpenSearch is, but what about the rest of that stuff?
+
+* **OpenSearch Operator for Kubernetes** controls and abstracts configuration of OpenSearch & OpenSearch Dashboards in the Kubernetes environment.
```

Comment:
Can we link back to each of these projects in their heading?

Reply:
10-4, will do.